Gems/Multiplayer/Code/Source/AutoGen/AutoComponent_Header.jinja

```diff
@@ -386,8 +386,6 @@ namespace {{ Component.attrib['Namespace'] }}
         {{ DeclareNetworkPropertyAccessors(Component, 'Autonomous', 'Authority', false)|indent(8) -}}
         {{ DeclareNetworkPropertyAccessors(Component, 'Autonomous', 'Authority', true)|indent(8) -}}
         {{ DeclareArchetypePropertyGetters(Component)|indent(8) -}}
-        {{ DeclareRpcInvocations(Component, 'Server', 'Authority', false)|indent(8) -}}
-        {{ DeclareRpcInvocations(Component, 'Server', 'Authority', true)|indent(8) -}}
```

Comment:
This is superfluous, we were declaring the invocation twice (here and on the component).

Reply:
Seems okay, I think the only thing that manifests due to this change is we'll have to call GetParent().<SendRpc>(...); But I am a huge fan of anything that reduces the amount of autogen code.